Gemfile.lock

```diff
@@ -529,7 +529,7 @@ GEM
     websocket-extensions (0.1.4)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.2.0)
+    zeitwerk (2.2.2)
```

Comment:
Should we change the minimum version in the gemspec?

Reply:
Good point. I'll do it.

Reply:
Done.